TestgenBookshopAPI/service/scripts/build.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# build.sh [repository]
+# Build the Bookshop images and push them to the specified repository.
+# You must already be logged in to the repository.
+# The default repository is the IBM Public Repository.
+
+cd $(dirname $0)
+script="$(basename $0)"
+repository=""
+
+USAGE="Usage:
+  ${script} --repository <repository> --api-key <api-key> [options]
```

Comment:
Since the repository is now required, it would usually be specified without the option; and since we don't actually use the API key, I think this could be simplified to:
```
${script} <repository>
```

Reply:
Done